metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -0,0 +1,191 @@
+Feature: Transitive release date control for child objects of a Study
+
+  Scenario Outline: Objects to which at least released study links should be accessible, and vice versa
+
+    # Create the common taxonomy
+    When I request POST taxonomies with 9606 for ID, Homo Sapiens for name and NONE for ancestors
+    And set the URL to TAXONOMY
+    Then the response code should be 201
+
+    # Create the common reference sequences
+    When I request POST /reference-sequences with JSON-like payload:
+    """
+          "name": "GRCh37",
+          "patch": "p2",
+          "accessions": ["GCA_000001407.3", "GCF_000001407.14"],
+          "type": "ASSEMBLY",
+          "taxonomy": "TAXONOMY"
+    """
+    And set the URL to REFERENCE_SEQUENCE
+    Then the response code should be 201
+
+    # Create two studies with the release date of "yesterday"
+    When I create a study with TAXONOMY for taxonomy and Study1 for accession
+    And set the URL to STUDY1
+    Then the response code should be 201
+    When I create a study with TAXONOMY for taxonomy and Study2 for accession
+    And set the URL to STUDY2
+    Then the response code should be 201
+
+    # Create two analyses: Study1 → Analysis1 and Study2 → Analysis2
+    When I create an analysis with Analysis1 for accession, REFERENCE_SEQUENCE for reference sequence and STUDY1 for study
+    And set the URL to ANALYSIS1
+    Then the response code should be 201
+    When I create an analysis with Analysis2 for accession, REFERENCE_SEQUENCE for reference sequence and STUDY2 for study
+    And set the URL to ANALYSIS2
+    Then the response code should be 201
+
+    # Create three files
+    When I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File1",
+        "version": 1
+        },
+      "hash": "Hash1",
+      "name": "File1",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    And set the URL to FILE1
+    Then the response code should be 201
+
+    When I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File2",
+        "version": 1
+        },
+      "hash": "Hash2",
+      "name": "File2",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    And set the URL to FILE2
+    Then the response code should be 201
+
+    When I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File3",
+        "version": 1
+        },
+      "hash": "Hash3",
+      "name": "File3",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    Then the response code should be 201
+    And set the URL to FILE3
```

Comment:
There is no `then` after this `and`, so no additional check is performed. Is that intentional?

Reply:
Not intentional, `Then` and `And` were accidentally swapped around. Now fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/b12ad94aa306c0e8eb84f967c15b5cb6baa5fc2c